### PR TITLE
fix:   Deleted files are visible on screen after clicking on delete button.

### PR DESCRIPTION
**Pull request by Rohini-Microsoft**
## Purpose
On click of delete button, pages refresh automatically and receives a notification file are deleted

Bug: https://dev.azure.com/CSACTOSOL/CSA%20Solutioning/_workitems/edit/5536

- [ ] Yes
- [x] No

## How to Test

1. Navigate to Chat with your data URL
2. Click on 'Delete Data' tab
3. Select files to Delete
4. Scroll down and select multiple checkboxes to delete
5. Delete files
6. Observe files are still on the screen but received notification "files are deleted"

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

image
****

### CODE REVIEW

File: code/backend/pages/03_Delete_Data.py [hunk 1]
```diff
@@ -73,6 +73,7 @@ def load_css(file_path):
                 )
                 if len(files_to_delete) > 0:
                     st.success("Deleted files: " + str(files_to_delete))
+                    st.experimental_rerun()
```
Comment (adamdougal) [inline code/backend/pages/03_Delete_Data.py:76]: Looks like this is deprecated in favour of `rerun`. https://docs.streamlit.io/develop/api-reference/execution-flow/st.experimental_rerun

Comment (adamdougal) [inline code/backend/pages/03_Delete_Data.py:76]: Does this essentially refresh the page to regenerate the list?

File: code/create_app.py [hunk 2]
```diff
@@ -427,6 +427,7 @@ def speech_config():
             if response.status_code == 200:
                 return {
                     "token": response.text,
+                    "key": speech_key,
```
Comment (adamdougal) [inline code/create_app.py:430]: This and all of the other changes in this PR, except `03_Delete_Data.py` look unrelated to the PR title and description. Are they supposed to be here?

**Review by adamdougal — CHANGES_REQUESTED** [review]
Can you update the PR title and explain the problem you're solving in the description? Thanks

File: code/backend/pages/03_Delete_Data.py (hunk 1 above)
Comment (Rohini-Microsoft) [reply code/backend/pages/03_Delete_Data.py:76]: Yes.

File: code/create_app.py (hunk 2 above)
Comment (Rohini-Microsoft) [reply code/create_app.py]: reverted changes

File: code/backend/pages/03_Delete_Data.py (hunk 1 above)
Comment (Rohini-Microsoft) [reply code/backend/pages/03_Delete_Data.py]: @adamdougal  I have made the changes. Can you please review it once.
